src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/EmptyLeaseTableSynchronizer.java

```diff
@@ -0,0 +1,84 @@
+package com.amazonaws.services.kinesis.clientlibrary.lib.worker;
+
+import com.amazonaws.services.kinesis.clientlibrary.types.ExtendedSequenceNumber;
+import com.amazonaws.services.kinesis.leases.impl.KinesisClientLease;
+import com.amazonaws.services.kinesis.leases.impl.Lease;
+import com.amazonaws.services.kinesis.model.Shard;
+import lombok.AllArgsConstructor;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static com.amazonaws.services.kinesis.clientlibrary.lib.worker.KinesisShardSyncer.StartingSequenceNumberAndShardIdBasedComparator;
+
+/**
+ * Class to help create leases when the table is initially empty.
+ */
+@AllArgsConstructor
+class EmptyLeaseTableSynchronizer implements LeaseSynchronizer {
+
+    private static final Log LOG = LogFactory.getLog(EmptyLeaseTableSynchronizer.class);
```

Comment:
Are we not using @Slf4j in 1.x?

Reply:
Nope, unfortunately 1.x still uses Apache logger